data/codeofconduct.md

```diff
@@ -1,38 +1,37 @@
 ## Introduction
 
-DACHFest is a community-driven conference for developers of all level of experience and background.
+DACHFest is a community-driven conference for developers of all experience levels and background.
 
 As a community-driven conference, we want everybody to feel welcome, regardless of gender, race, religion, sexual orientation, disabilites or physical appearance. We do not tolerate harassment of any kind.
 
-To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and volunteers at any DevFest Berlin event are required to conform to the following Code of Conduct. Organizers will enforce this code throughout the event.
+_Everyone_ (including speakers and staff) is required to conform to the following Code of Conduct. Organizers and volunteers will enforce this code throughout the event.
```

Comment:
To make it 100% clear, I'd add that sponsors have to comply with the code of conduct too.

Reply:
Done!